altair/examples/airports_count.py

```diff
@@ -0,0 +1,47 @@
+"""
+Locations of US Airports
+========================
+This is a layered geographic visualization that shows the positions of US
+airports on a background of US states.
+"""
+# category: maps
+import altair as alt
+from vega_datasets import data
+
+airports = data.airports()
+airports_counts = airports[['state','latitude', 'longitude']]
+
+# Take the mean as the position of state
+agg = {'latitude': 'mean',
+         'longitude': 'mean'
+                  }
+airports_counts = airports.groupby(['state']).agg(agg) 
+agg = {'latitude': 'count',
+      
+                  }
+
+airports_counts['Number of Airports'] =  airports.groupby(['state']).agg(agg)
```

Comment:
This would be easier to follow if you did all three aggregates in a single operation.

Reply:
I did it because the earlier aggregation is using the same column, while seperating this aggregration allows me to assign it to a new column in one line. 

I would love to know is there a cleaner pandas way to do so? I agree using transform_aggregate is cleaner in this case.

```
agg = {'latitude': 'mean',
         'longitude': 'mean'
                  }
```

Reply:
Your third aggregate is count, so it can be computed on any column. For example, this gives the same result:
```python
airports_counts = airports.groupby('state').agg({
    'latitude': 'mean',
    'longitude': 'mean',
    'name': 'count'
}).rename({'name': 'Number of Airports'}, axis=1)
```
But overall I like the transform approach below better.

Reply:
Didn't know we can pass count directly without an actual column. I try to use  'latitude': ['mean','count'] before but that result in multi-index columns and is less easy to rename.

Reply:
You do have to pass a column; in this case I passed the column called ``"name"``, which has the same count as latitude (or as any of the other columns)